stripe/test/test_resources.py

```diff
@@ -208,6 +208,60 @@ def test_retrieve(self):
         self.assertResponse(res)
 
 
+class ListObjectTests(StripeApiTestCase):
```

Comment:
Another option would be to add an `iter()` method to list objects, making it clearer that it will return all objects. Thoughts?